src/async.cpp

```diff
@@ -179,7 +181,20 @@ void my_plugin::async_thread_loop(std::unique_ptr<falcosecurity::async_event_han
 	}
 
 exit:
-	SPDLOG_INFO("Async thread terminated");
+	SPDLOG_INFO("Stop FUSE fs thread");
+}
+
+void my_plugin::diff_notify(std::unique_ptr<falcosecurity::async_event_handler> h) noexcept {
+	SPDLOG_INFO("Start diff notify thread");
+	while(!m_diff_notify_thread_quit.load()) {
+		// -1 should be a valid tid for async events
+		generate_async_event(h, ASYNC_EVENT_DIFF_NAME, -1, "root");
+		std::unique_lock<std::mutex> l(m_diff_mu);
+		m_diff_cv.wait_for(l, std::chrono::seconds(DIFF_NOTIFY_INTERVALS), [this] {
+			return m_diff_notify_thread_quit.load();
+		});
```

Comment:
nit: you could avoid using an atomic by doing the following
```suggestion
        bool done = false;
	while(done) {
		// -1 should be a valid tid for async events
		generate_async_event(h, ASYNC_EVENT_DIFF_NAME, -1, "root");
		std::unique_lock<std::mutex> l(m_diff_mu);
		done = m_diff_cv.wait_for(l, std::chrono::seconds(DIFF_NOTIFY_INTERVALS), [this] {
			return m_diff_notify_thread_quit;
		});
```